### PR TITLE
fix userSelectPallete gaps and footer border theming

### LINKED ISSUE #186: [BUG] : Add separator to pallete to differentiate between categories
### Has this bug been raised before?

- [X] I have checked "open" AND "closed" issues and this is not a duplicate

### Description

title i am talking about>
Screenshot 2024-10-18 110703

i thing line is a bit confusing to users. Intead this we can just add two dropdowns for timilimits and text lengths.

### Steps to Reproduce

1.Go to homepage 
2. see top titlte and try to click each words

### Screenshots

_No response_

### Do you want to work on this issue?

Yes

### If "yes" to above, please explain how you would technically implement this

I wiil add two dropdowns intead of that line and a title above it(some text to look ui good).

### PR DESCRIPTION
**Pull request by Sahil-Gupta584**
#### THIS IS A PULL REQUEST TEMPLATE! 

⚠️ This helps you complete a pull request. Please follow the instructions below. Reminder that you are bound to abide by the [Code of Conduct](/CODE_OF_CONDUCT.md) ⚠️ 

### Submission Checklist

---

Please complete the below checklist. Failure to follow these instructions may result in your pull request being closed without review.

### What issue does this PR closes?

closes #186 , #99 

### Description of file changes

1.for footer changes i have modifed tailwind config a bit,added borderColor along side background,foreground variables

### Is this PR still in progress or completed?
- [ ] This PR is still a work in progress (WIP)
- [X] This PR is completed and ready for review
- [ ] Did you edit the package.json or pnpm.yaml? If yes, explain reason for modification and add the `dependencies` label to this PR. <br>

### CODE REVIEW

File: styles/globals.css
Comment (JhohannesK) [inline styles/globals.css:1]: remove the `--border-color``.

File: tailwind.config.js
Comment (JhohannesK) [inline tailwind.config.js:1]: remove this also

File: components/Footer.tsx
Comment (JhohannesK) [inline components/Footer.tsx:1]: Use color from the existing variables...

**Review by JhohannesK — CHANGES_REQUESTED** [review]
I don't what's actually wrong, but the PR fails to load the actual font for the application.

**Comment by Sahil-Gupta584** [comment]
@JhohannesK i have modified code as you said and use input variable color for footer

**Pull request merged**